lms/djangoapps/commerce/tests/test_utils.py

```diff
@@ -16,3 +18,54 @@ def test_log_message(self, mock_log):
         # key-value pairs ordered alphabetically by key.
         message = 'foo: bar="baz", qux="quux"'
         self.assertTrue(mock_log.info.called_with(message))
+
+
+class EcommerceServiceTests(TestCase):
+    """Tests for the EcommerceService helper class."""
+    SKU = 'TESTSKU'
+
+    def setUp(self):
+        CommerceConfiguration.objects.create(
+            checkout_on_ecommerce_service=True,
+            single_course_checkout_page='/test_basket/'
+        )
+        super(EcommerceServiceTests, self).setUp()
+
+    def test_is_enabled(self):
+        """Verify that is_enabled() returns True."""
```

Comment:
Seeing your other tests, this docstring needs to be more informative: Verify `is_enabled() returns True when not used for a microsite.